spec/features/05_show_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "Authenticated user" do
+  let!(:populate_db) do
+    user1 = FactoryGirl.create(:user)
+    @movies2 = FactoryGirl.create_list(:movie, 5)
+  end
+
+  describe "the signin process" do
+    before :each do
+      visit new_user_session_path
+      fill_in "Email", with: user1.email
+      fill_in "Password", with: user1.password
+      click_button "Sign In"
+      expect(page).to have_content("Welcome to Cinephile!")
```

Comment:
it would be a good idea (after this merge) to pull this out and make it a method in your spec_helper. It could be named something like `user_signs_in` or something like that